src/main/scala/edu/ie3/simona/io/runtime/RuntimeEventLogSink.scala

```diff
@@ -76,8 +77,11 @@ final case class RuntimeEventLogSink(
     )
   }
 
-  private def roundDuration(duration: Double): Double = {
-    roundAt(5)(duration / 1000)
+  private def convertDuration(duration: Double): String = {
+    val hh = ((duration / 1000 / 60) % 24).toInt
```

Comment:
Please leave out the `% 24`, as we want to see the full hour amount :) Also, I think another division by 60 is necessary to get to hours.